src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -63,8 +69,16 @@ def __init__(self, config, device_id: int = 0, obs_spaces=None, action_spaces=No
                 config,
                 config.AGENT.SKILLS.NAV_TO_OBJ,
                 device_id=device_id,
-                obs_spaces=obs_spaces,
-                action_spaces=action_spaces,
+                obs_spaces=None,
+                action_spaces=None,
+            )
+        if config.AGENT.SKILLS.NAV_TO_REC.type == "rl":
```

Comment:
Adds checks for skips here and before so that the agent is not created unless asked for

Reply:
Done